shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -326,6 +379,75 @@ impl CassandraConnection {
                 let query = session.prepare(query).await.unwrap();
                 PreparedQuery::CdrsTokio(query)
             }
+            Self::Scylla { session, .. } => {
+                let mut prepared = session.prepare(query).await.unwrap();
+                prepared.set_tracing(true);
+                PreparedQuery::Scylla(prepared)
+            }
+        }
+    }
+
+    #[allow(dead_code)]
+    pub async fn execute_prepared_with_tracing(
```

Comment:
either name this something like: `execute_prepared_returning_coordinator_node`
Or return something like:
```rust
struct TracingResults {
    coordinator_node: IpAddr,
}
```

Depends on if you expect to need to expose more tracing values in the future

Reply:
I'll go with #1, leave the second for a future improvement if required.